docs/components/query.md

```diff
@@ -28,49 +29,41 @@ Thanos Querier essentially allows to aggregate and optionally deduplicate multip
-    * e.g [OpenTSDB](../integrations.md#opentsdb)
+  * e.g [OpenTSDB](../integrations.md#opentsdb)
 
 Thanks to that, you can run queries (manually, from Grafana or via Alerting rule) that aggregate metrics from mix of those sources.
 
 Some examples:
 
-* `sum(cpu_used{cluster=~"cluster-(eu1|eu2|eu3|us1|us2|us3)", job="service1"})` that will give you sum of CPU used inside all listed clusters for service `service1`. This will work
-even if those clusters runs multiple Prometheus servers each. Querier will know which data sources to query.
+* `sum(cpu_used{cluster=~"cluster-(eu1|eu2|eu3|us1|us2|us3)", job="service1"})` that will give you sum of CPU used inside all listed clusters for service `service1`. This will work even if those clusters runs multiple Prometheus servers each. Querier will know which data sources to query.
 
 * In single cluster you shard Prometheus functionally or have different Prometheus instances for different tenants. You can spin up Querier to have access to both within single Query evaluation.
 
 ### Run-time deduplication of HA groups
 
-Prometheus is stateful and does not allow replicating its database. This means that increasing high availability by running multiple Prometheus replicas is not very easy to use.
-Simple load balancing will not work as for example after some crash, replica might be up but querying such replica will result in small gap during the period it was down. You have a
- second replica that maybe was up, but it could be down in other moment (e.g rolling restart), so load balancing on top of those is not working well.
+Prometheus is stateful and does not allow replicating its database. This means that increasing high availability by running multiple Prometheus replicas is not very easy to use. Simple load balancing will not work as for example after some crash, replica might be up but querying such replica will result in small gap during the period it was down. You have a second replica that maybe was up, but it could be down in other moment (e.g rolling restart), so load balancing on top of those is not working well.
 
 Thanos Querier instead pulls the data from both replicas, and deduplicate those signals, filling the gaps if any, transparently to the Querier consumer.
 
 ## Metric Query Flow Overview
 
 <img src="../img/querier.svg" class="img-fluid" alt="querier-steps" />
 
-Overall QueryAPI exposed by Thanos is guaranteed to be compatible with [Prometheus 2.x. API](https://prometheus.io/docs/prometheus/latest/querying/api/).
-The above diagram shows what Querier does for each Prometheus query request.
+Overall QueryAPI exposed by Thanos is guaranteed to be compatible with [Prometheus 2.x. API](https://prometheus.io/docs/prometheus/latest/querying/api/). The above diagram shows what Querier does for each Prometheus query request.
 
 See [here](https://thanos.io/tip/thanos/service-discovery.md/) on how to connect Querier with desired StoreAPIs.
 
-<!--- TODO explain steps  --->
```

Comment:
This TODO was also removed. Looks like mdox is stripping out raw HTML.

Reply:
This is fixed [here](https://github.com/bwplotka/mdox/pull/46)! mdox now preserves in-md HTML. 🙂